### PR TITLE
moved dependency fixes from protected to here

### PR DESCRIPTION
**Pull request by akiezun**
these are moved over from protected to public. Without them, even count reads does not work after we added mllib.

### CODE REVIEW

**Comment by akiezun** [comment]
@droazen can you review? Look at protected build file

File: build.gradle [hunk 1]
```diff
@@ -156,6 +156,19 @@ dependencies {
         exclude module: 'kryo' // use Spark's version
     }
```
Comment (droazen) [inline build.gradle]: Do you need this as well (from protected)?

    // Comment the next line to disable native code proxies in Spark MLLib
    compile('com.github.fommil.netlib:all:1.1.2')

File: build.gradle [hunk 2]
```diff
@@ -156,6 +156,19 @@ dependencies {
         exclude module: 'kryo' // use Spark's version
```
Comment (droazen) [inline build.gradle]: protected uses a different version of this dependency (0.26) -- will there be a conflict?

**Comment by droazen** [comment]
@akiezun Can you please test gatk-protected with a local snapshot of this branch, and confirm that protected still builds and tests pass?

**Comment by droazen** [comment]
Back to @akiezun

**Comment by akiezun** [comment]
it won't. Protected is way behind and needs an urgent revup

**Comment by droazen** [comment]
@akiezun In that case, merge when ready and we'll have to deal with the breakage in protected separately. This is going to cause issues for people who need to test out public snapshots of branches with protected, so we should fix soon.

File: build.gradle (hunk 1 above)
Comment (akiezun) [inline build.gradle]: i don't know. Let's assume yes

File: build.gradle (hunk 2 above)
Comment (akiezun) [inline build.gradle]: protected will get rid of this dependency once they rev up

**Pull request merged**